.github/workflows/ci.yml

```diff
@@ -15,3 +15,5 @@ jobs:
           php-version: ${{ matrix.php-version }}
       - run: composer install --prefer-dist --no-interaction
       - run: vendor/bin/php-cs-fixer fix --ansi --dry-run --using-cache=no --verbose
+      - run: wget https://phar.phpunit.de/phpunit-9.6.phar
```

Comment:
**praise**: 👏🏼 for adding tests

Reply:
Indeed. Phpunit is now added through composer. :ok_hand: